.github/workflows/lint.yml

```diff
@@ -41,6 +41,17 @@ jobs:
           # use this.
           warnings: "-native-cc"
 
+  yapf:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checking out repository
+        uses: actions/checkout@v2
+      - name: Running yapf
+        run: |
+          python3 -m pip install yapf
+          printf "\nIf the lint below fails it can be fixed by running \n  python3 -m yapf --verbose --in-place --exclude='*/third_party/*' --recursive ../iree\n\n"
+          python3 -m yapf --diff --exclude='*/third_party/*' --recursive ../iree
```

Comment:
Why not just `.` instead of `../iree`? Does this have to do with the --exclude parameter? I'd rather just list two exclusion patterns. Even better though would be to add a .yapfignore to the root directory (although Scott will get upset about more root files :grin:). This should make the instructions for users simpler and also affect people's local setups.

Reply:
I was using `../iree` because otherwise the command would hang for several minutes, but it looks like the exclude wasn't working properly. If it's not too annoying to have a `.yapfignore` file it would be nice to be able to just run `python3 -m yapf --diff -r .`

@ScottTodd?

Reply:
This isn't relevant anymore since we switched to formatting git diffs.